cypher/standards/conditions.clsp

```diff
@@ -79,9 +79,12 @@
     (r (r condition)))
 
   ; AssertCoinAnnouncement
-
+;message here: (sha256 RING_MORPH_BYTE this_coin_id this_subtotal)
   (defun-inline cypher.standards.conditions.AssertCoinAnnouncement (coin_id message)
     (cypher.standards.conditions.Condition ASSERT_COIN_ANNOUNCEMENT (sha256 coin_id message)))
+;message here: (sha256tree (list this_coin_id this_subtotal))
```

Comment:
This should be a CAT-level function. Standards should be just generic conditions.